### PR TITLE
Add isPositiveNumber method to NumericHelper

### PR DESCRIPTION
**Pull request by arogachev**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  |

Extracted from https://github.com/yiisoft/validator/pull/151#discussion_r792360314. 

> :warning: A new release (2.1.0) is required for https://github.com/yiisoft/validator/pull/151.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,7 +18,7 @@
 ## 1.2.0 January 22, 2021
 
 - Enh #62: Add method `StringHelper::split()` that split a string to array with non-empty lines (vjik)
-- Enh #63: Add method `NumericHelper::isInteger()` that checks whether the given string is an integer number (vjik)
+- Enh #63: Add method `NumericHelper::isInteger()` that checks whether the given value is an integer number (vjik)
```
Comment (samdark) [inline CHANGELOG.md:21]:
```suggestion
- Enh #63: Add method `NumericHelper::isInteger()` that checks whether the given value represents an integer number (vjik)
```

File: src/NumericHelper.php [hunk 2]
```diff
@@ -73,7 +73,7 @@ public static function normalize($value): string
     }
 
     /**
-     * Checks whether the given string is an integer number.
+     * Checks whether the given value is an integer number.
```
Comment (samdark) [inline src/NumericHelper.php:76]:
```suggestion
     * Checks whether the given value represents an integer number.
```

File: src/NumericHelper.php [hunk 3]
```diff
@@ -83,4 +83,21 @@ public static function isInteger($value): bool
     {
         return filter_var($value, FILTER_VALIDATE_INT) !== false;
     }
+
+    /**
+     * Checks whether the given string is a positive integer number.
```
Comment (samdark) [inline src/NumericHelper.php:88]:
```suggestion
     * Checks whether the given string represents a positive integer number.
```

File: CHANGELOG.md (hunk 1 above)
Comment (arogachev) [reply CHANGELOG.md]: :heavy_check_mark:

File: src/NumericHelper.php (hunk 2 above)
Comment (arogachev) [reply src/NumericHelper.php]: :heavy_check_mark:

File: src/NumericHelper.php (hunk 3 above)
Comment (arogachev) [reply src/NumericHelper.php]: :heavy_check_mark:

**Comment by arogachev** [comment]
By the way, according to https://github.com/yiisoft/docs/blob/master/000-packages.md#versions, this and the previous change should be part of 2.1.0, not 2.0.1. Updated CHANGELOG.

File: src/NumericHelper.php [hunk 4]
```diff
@@ -83,4 +83,21 @@ public static function isInteger($value): bool
...
     }
+
+    /**
+     * Checks whether the given string represents a positive integer number.
+     *
+     * @param string $string
+     *
+     * @return bool
+     */
+    public static function isPositiveInteger(string $string): bool
```
Comment (vjik) [inline src/NumericHelper.php:94]: Check only strings is not consistent to the `isInteger` method with mixed argument.

Maybe also use mixed argument and goal of this method will be to check that type casting value to integer will give positive integer?

Comment (samdark) [reply src/NumericHelper.php:94]: `+1` string will be cast to positive integrer.

**Comment by arogachev** [comment]
It's no longer needed in validator (a different approach is used). After a discussion, we decided to delay it until it's needed more.

**Pull request closed without merging**